docs/book/features/volume_health.md

```diff
@@ -0,0 +1,85 @@
+# vSphere CSI Driver - Volume Health
+
+Volume Health feature provides the health status of persistent volume. Volume Health is supported for Block Volumes in vSphere with Kubernetes Cluster and Tanzu Kubernetes Grid Cluster - vSphere 7.0u1.
```

Comment:
How about this?

`Volume Health is supported for Block Volumes in vSphere with Kubernetes Cluster and Tanzu Kubernetes Grid Cluster releases for vSphere 7.0 Update 1.`